### ISSUE TITLE
Upload a version on hackage

### ISSUE THREAD

**Issue by guibou** [meteogrid/bindings-gdal#16]
Could you upload a version on hackage? It will ease the use of `bindings-gdal` with tools such as `cabal`, `stack` or `nix`.

**Comment by albertov** [comment]
Yeah, that's something that's long overdue. I'll try to do it in the next few days. I need to first make sure the `.cabal` has proper bounds for all dependencies which might take a while (I'm using nix to build this in my projects so got a bit lazy with that)

**Comment by Sintrastes** [comment]
@albertov I'd still be very much interested this if you get the chance.
